<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>901</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QCustomPlot" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>881</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>0</y>
      <width>221</width>
      <height>33</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Function + 1 method</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Function + 2 method</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Function + 1 and 2 methods</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Error plots</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>0</y>
      <width>113</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>0</y>
      <width>113</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>0</y>
      <width>113</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>30</y>
      <width>341</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Left boundary           Right boundary       Number of points</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>50</y>
      <width>391</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Standart:            -5                                     5                                  6</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>0</y>
      <width>121</width>
      <height>33</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Sin(x)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>From file</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_3">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>71</width>
      <height>33</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>100 %</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>150 %</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>200 %</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>75 %</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>50 %</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Zoom:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>901</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Press 3 to change zoom</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuPress_F1_for_change_input_function">
    <property name="title">
     <string>Press 1 to change input function</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuPress_F2_to_change_type_of_showed_plots">
    <property name="title">
     <string>Press 2 to change type of showing plots</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuPress_4_to_multiply_amount_of_points_by_2">
    <property name="title">
     <string>Press 4 to multiply amount of points by 2</string>
    </property>
   </widget>
   <addaction name="menuPress_F1_for_change_input_function"/>
   <addaction name="menuPress_F2_to_change_type_of_showed_plots"/>
   <addaction name="menu"/>
   <addaction name="menuPress_4_to_multiply_amount_of_points_by_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
